Amazon.IonObjectMapper/IonSerializer.cs

```diff
@@ -365,7 +365,8 @@ internal object Deserialize(IIonReader reader, Type type, IonType ionType)
             if (ionType == IonType.Blob)
             {
                 if (annotations.Any(s => s.Equals(IonGuidSerializer.ANNOTATION))
-                    || typeof(Guid).IsAssignableFrom(type))
+                    || typeof(Guid).IsAssignableFrom(type)
+                    || typeof(Nullable<Guid>).IsAssignableFrom(type))

```

Comment:
could use `Guid?` shorthand for `Nullable<Guid>`

Reply:
Updated